static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -341,6 +338,16 @@ const ContentRow = styled('div')`
   gap: ${space(0.75)};
 `;
 
+const MemberEmail = styled('div')`
+  display: block;
+  max-width: 150px;
```

Comment:
Is this still going to be an issue on smaller screen sizes?

Reply:
the whole table looks weird on the modal when you shrink the screen size...